src/ring_r.c

```diff
@@ -1,4 +1,5 @@
 #include "ring_r.h"
+#include <float.h>
```

Comment:
Had to look up with this did - and it's for DBL_EPSILON on 345 - but I wonder how it worked before without the include...?

Reply:
Oh right - without R_STRICT_HEADERS it somehow resolved, but when that's added in, it doesn't...